frontend/src/scss/PrimeStyles.scss

```diff
@@ -130,6 +130,24 @@
   background-color: color($theme-color-prime-red-lightest);
 }
 
+.prime-radio--horizontal {
```

Comment:
N.B.: USWDS used to have the `horizontal` option but advised only to use it for two choices. Now they are saying always do it vertically. I tend to agree that two options with short names like Yes/No are best done horizontally so no objections here.